test/test_timeout.rb

```diff
@@ -109,6 +109,7 @@ def test_nested_timeout_which_error_bubbles_up
     end
 
     assert_equal 'inner message', e.message
+    assert_equal 'inner message', raised_exception.message
```

Comment:
```suggestion
    assert_equal 'inner message', raised_exception.message
```
I meant this. Since there is a `raised_exception` instead of `e`, which "leaks" to the method body